content/en/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -78,105 +76,15 @@ Finally, kube-scheduler assigns the Pod to the Node with the highest ranking.
-   {{< glossary_tooltip term_id="statefulset" >}} or
-   {{< glossary_tooltip term_id="replica-set" >}}.
-
-- `InterPodAffinityPriority`: Computes a sum by iterating through the elements
-  of weightedPodAffinityTerm and adding “weight” to the sum if the corresponding
-  PodAffinityTerm is satisfied for that node; the node(s) with the highest sum
-  are the most preferred.
-
-- `LeastRequestedPriority`: Favors nodes with fewer requested resources. In other
-  words, the more Pods that are placed on a Node, and the more resources those
-  Pods use, the lower the ranking this policy will give.
-
-- `MostRequestedPriority`: Favors nodes with most requested resources. This policy
-  will fit the scheduled Pods onto the smallest number of Nodes needed to run your
-  overall set of workloads.
-
-- `RequestedToCapacityRatioPriority`: Creates a requestedToCapacity based ResourceAllocationPriority using default resource scoring function shape.
-
-- `BalancedResourceAllocation`: Favors nodes with balanced resource usage.
-
-- `NodePreferAvoidPodsPriority`: Prioritizes nodes according to the node annotation
-  `scheduler.alpha.kubernetes.io/preferAvoidPods`. You can use this to hint that
-  two different Pods shouldn't run on the same Node.
-
-- `NodeAffinityPriority`: Prioritizes nodes according to node affinity scheduling
-   preferences indicated in PreferredDuringSchedulingIgnoredDuringExecution.
-   You can read more about this in [Assigning Pods to Nodes](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/).
-
-- `TaintTolerationPriority`: Prepares the priority list for all the nodes, based on
-  the number of intolerable taints on the node. This policy adjusts a node's rank
-  taking that list into account.
-
-- `ImageLocalityPriority`: Favors nodes that already have the
-  {{< glossary_tooltip text="container images" term_id="image" >}} for that
-  Pod cached locally.
-
-- `ServiceSpreadingPriority`: For a given Service, this policy aims to make sure that
-  the Pods for the Service run on different nodes. It favours scheduling onto nodes
-  that don't have Pods for the service already assigned there. The overall outcome is
-  that the Service becomes more resilient to a single Node failure.
-
-- `CalculateAntiAffinityPriorityMap`: This policy helps implement
-  [pod anti-affinity](/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity).
-
-- `EqualPriorityMap`: Gives an equal weight of one to all nodes.
+1. [Scheduling Policies](/docs/reference/kube-scheduler/policies) allows you to
+  configure *Predicates* for filtering and *Priorities* for scoring.
+1. [Scheduling Profiles](/docs/reference/kube-scheduler/profiles) allows you to
+  configure Plugins that implement different scheduling stages, including:
+  `QueueSort`, `Filter`, `Score`, `Bind`, `Reserve`, `Permit` and others. You
+  can also configure the kube-scheduler to run different profiles.
```

Comment:
Fixing broken hyperlinks; also revising for grammar and to match website style guide.
```suggestion
1. [Scheduling Policies](/docs/reference/scheduling/policies/) allow you to
  configure _predicates_ for filtering and _priorities_ for scoring.
1. [Scheduling Profiles](/docs/reference/scheduling/profiles/) allow you to
  configure Plugins that implement different scheduling stages, including:
  `QueueSort`, `Filter`, `Score`, `Bind`, `Reserve`, `Permit`, and others. You
  can also configure the kube-scheduler to run different profiles.
```